narratives/ncov_sit-rep_2020-03-20.md

```diff
@@ -0,0 +1,2453 @@
+<!-- This is left-side text 8 -->
+# [Epidemics become evident ~4-8 weeks after introduction](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=num_date&d=tree&f_division=Washington&label=clade:B1&p=full)
+
+It may seem as though COVID-19 has suddenly exploded -- for many people, this outbreak may have gone from an abstract concern in other parts of the world to something that severely impacts our daily lives, seemingly overnight.
+<br><br>
+But, keep in mind that this has been brewing for awhile. It's not that the virus is being introduced to new places more often.
+Rather, it's that introductions that happened weeks ago started localized outbreaks that we're now becoming aware of.
+<br><br>
+Areas that aren't yet aware of local outbreaks should prepare now and start surveillance testing.  
+
+<!-- This is the right-side text -->
+
+```auspiceMainDisplayMarkdown
+<img src="https://github.com/nextstrain/ncov/raw/master/figures/local-spark-expansion.jpeg" width="70%">
+
+"COVID-19 in Wuhan went from an index case in ~Nov 2019 to several thousand cases by mid-Jan 2020, thus going from initial seeding event to widespread local transmission in the span of ~10 weeks. We believe that international seeding events started to occur in mid-Jan. Thus we have a critical ~10 weeks from then to late-March to contain these nascent outbreaks before they become sizable." [- Trevor Bedford](https://twitter.com/trvrb/status/1226241284207038464), Feb 2020
+```
+
+<!-- ############ SLIDE BREAK ############# -->
+# [Once introduced, epidemics become evident ~4-8 weeks later](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington&label=clade:B1&p=grid&r=location)
+
+We can see a clear example of this in Washington state.
+Mousing over the internal nodes at the root of the clade tells us that the virus was likely introduced once to this area in early- to mid-January 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)).
+<br><br>
+Now, in mid-March (~8 weeks later), we see that a local outbreak is in full swing in this area. This is consistent with a doubling rate of about 4 days, assuming an exponentially growing population.  
+<br>
+<img src="https://github.com/nextstrain/ncov/raw/master/figures/wa_doubling-rate_2020-03-19.png" width="70%">
+
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 9-->
+# [How outbreaks grow: an example](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington,Utah&label=clade:B1&p=grid)
+
+We can also see early evidence that 'sparks' from this cluster have led to introductions in Utah. Following this pattern, we expect to see a local outbreak in Utah pick up within the next 4 weeks.
+<br><br>
+This is just one example. Most importantly, areas that aren't yet aware of local outbreaks should prepare now and start surveillance testing.  
```

Comment:
Between "...example. Most..." add:
"There may have been other introductions into Utah (or other areas) that we cannot see on the tree because they were not sequenced."